src/components/screens/new-questionnaire-screen.tsx

```diff
@@ -66,24 +70,31 @@ const NewQuestionnaireScreen = () => {
    * @param list of QuestionOptions
    */
   const handleAddQuestion = (questionText: string, options: QuestionOption[]) => {
-    setOptions((prevQuestions: Question[]) => [...prevQuestions, { questionText, options }]);
+    setQuestionnaire((prevQuestionnaire) => ({
+      ...prevQuestionnaire,
+      options: [
```

Comment:
Maybe it's a bit strange options are called options, as to me it implies its just the options for a question. Should it be called questions, as it includes the question text and the options? What do you think?

Reply:
I realize this would also require a change to the specs and the lambdas, so maybe it's not necessary. Perhaps if you find yourself with spare time as I think the naming would be clearer, especially as the Type of options in Questionnaire is Questions (but it's called Questions), and within the questions type we have the Options type which is a label and a value.....

So actually this should be changed as the naming is quite confusing

Reply:
Naming is confusing, I agree with that. Perhaps this could be done as seperate fix PR.